packages/shorebird_cli/lib/src/commands/release_new/release_new_command.dart

```diff
@@ -0,0 +1,420 @@
+
+        logger
+          ..success('✅ Published Release ${release.version}!')
+          ..info(releaser.postReleaseInstructions);
+
+        printPatchInstructions(
+          releaser: releaser,
+          releaseVersion: releaseVersion,
+          releaseType: releaser.releaseType,
+          flavor: flavor,
+          target: target,
+        );
+      },
+      values: {
+        shorebirdEnvRef.overrideWith(() => releaseFlutterShorebirdEnv),
+      },
+    );
+  }
+
+  /// Determines which Flutter version to use for the release. This will be
+  /// either the version specified by the user or the version provided by
+  /// [shorebirdEnv]. Will exit with code 70 if the version specified by the
+  /// user is not found/supported.
+  Future<String> resolveTargetFlutterRevision() async {
+    if (flutterVersionArg != null) {
+      final String? revision;
+      try {
+        revision = await shorebirdFlutter.getRevisionForVersion(
+          flutterVersionArg!,
+        );
+      } catch (error) {
+        logger.err(
+          '''
+Unable to determine revision for Flutter version: $flutterVersionArg.
+$error''',
+        );
+        exit(ExitCode.software.code);
+      }
+
+      if (revision == null) {
+        final openIssueLink = link(
+          uri: Uri.parse(
+            'https://github.com/shorebirdtech/shorebird/issues/new?assignees=&labels=feature&projects=&template=feature_request.md&title=feat%3A+',
+          ),
+          message: 'open an issue',
+        );
+        logger.err(
+          '''
+Version $flutterVersionArg not found. Please $openIssueLink to request a new version.
+Use `shorebird flutter versions list` to list available versions.
+''',
+        );
+        exit(ExitCode.software.code);
+      }
+
+      return revision;
+    }
+
+    return shorebirdEnv.flutterRevision;
+  }
+
+  /// Asserts that a release with version [version] can be released using
+  /// flutter revision [flutterRevision]. If a release has already been
+  /// published with the given [version] for the platform [releasePlatform], or
+  /// if a release already exists with [version] but was compiled with a
+  /// different Flutter revision, an error will be thrown.
+  Future<void> assertVersionIsReleasable({
+    required String version,
+    required String flutterRevision,
+    required ReleasePlatform releasePlatform,
+  }) async {
+    final existingRelease = await codePushClientWrapper.maybeGetRelease(
+      appId: appId,
+      releaseVersion: version,
+    );
+
+    if (existingRelease != null) {
+      codePushClientWrapper.ensureReleaseIsNotActive(
+        release: existingRelease,
+        platform: releasePlatform,
+      );
+
+      // All artifacts associated with a given release must be built
+      // with the same Flutter revision.
+      if (existingRelease.flutterRevision != flutterRevision) {
+        // All artifacts associated with a given release must be built
+        // with the same Flutter revision.
```

Comment:
```suggestion
```
I guess we don't need this comment? It is equals to the on above

Reply:
Good catch, done